bindings/java/c/host.c

```diff
@@ -18,11 +18,20 @@ static JNIEnv* attach()
     return jenv;
 }
 
+static jbyteArray copyDataToJava(JNIEnv* jenv, const void* ptr, size_t size)
```

Comment:
Seems to be inconsistent with other function names. I not sure we should aim for consistent naming convention here.

Reply:
It is somewhat consistent with the JNI helpers though. Perhaps follow those?